src/components/disease/diseaseComparisonRibbon.js

```diff
@@ -128,23 +139,35 @@ class DiseaseComparisonRibbon extends Component {
         </div>
 
         <HorizontalScroll>
-          <div className='text-nowrap'>
-            <GenericRibbon
-              categories={summary.data.categories}
-              colorBy={COLOR_BY.CLASS_COUNT}
-              hideFirstSubjectLabel
-              itemClick={this.onDiseaseGroupClicked}
-              newTab={false}
-              selected={selectedBlock}
-              selectionMode={SELECTION.COLUMN}
-              subjectLabel={subject => <RibbonGeneSubjectLabel gene={subject} isFocusGene={subject.id === geneId} />}
-              subjectLabelPosition={POSITION.LEFT}
-              subjects={summary.data.subjects}
-            />
+          <div className='text-nowrap' style={{'width' : '1100px' }} >
```

Comment:
Is setting a fixed width here necessary?

Reply:
Since you already have the HorizontalScroll component and text-nowrap class, no, but otherwise the ribbon takes the width of the parent component and could wrap